bastion/src/context.rs

```diff
@@ -362,6 +364,151 @@ impl BastionContext {
             pending!();
         }
     }
+
+    /// Returns [`RefAddr`] of the current `BastionContext`
```

Comment:
Please add an example.

Reply:
it's used by a macro that's because it needs to be public. It's not intended to be used by an end-user.

Reply:
Can you, then, remove the doc and put `#[doc(hidden)]` instead? :)